frontend/src/app/main/data/workspace/selection.cljs

```diff
@@ -114,7 +114,11 @@
                (rx/buffer-time 100)
                (rx/map last)
                (rx/pipe (rxo/distinct-contiguous))
-               (rx/map #(select-shapes-by-current-selrect preserve? ignore-groups?))))
+               (rx/with-latest-from ms/keyboard-mod)
```

Comment:
You can pass multiple streams at once to the with-latest-from

Reply:
OK